framework-plugins/lisk-framework-chain-connector-plugin/src/types.ts

```diff
@@ -34,6 +34,67 @@ export interface ChainConnectorInfo {
 	blockHeaders: chain.BlockHeader[];
 	aggregateCommits: AggregateCommit[];
 	validatorsHashPreimage: ValidatorsData[];
+	crossChainMessages: CCMsg[];
+}
+
+export interface Inbox {
+	appendPath: Buffer[];
+	size: number;
+	root: Buffer;
+}
+
+export interface InboxJSON {
+	appendPath: string[];
+	size: number;
+	root: string;
+}
+
+export interface Outbox {
+	appendPath: Buffer[];
+	size: number;
+	root: Buffer;
+}
+
+export interface OutboxJSON {
+	appendPath: string[];
+	size: number;
+	root: string;
+}
+
+export interface MessageFeeTokenID {
+	chainID: Buffer;
+	localID: Buffer;
+}
+
+export interface MessageFeeTokenIDJSON {
+	chainID: string;
+	localID: string;
```

Comment:
These types can also be imported directly from [here](https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/interoperability/types.ts)

Reply:
exported